src/js/disability-benefits/686/config/form.js

```diff
@@ -1,42 +1,140 @@
 // import fullSchema from 'vets-json-schema/dist/686-schema.json';
+import _ from 'lodash';
```

Comment:
We try to use `lodash/fp` for the forms, generally, since it avoid mutating the arguments you pass.